omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -22,9 +22,6 @@
 
 	<!-- Add here beans related to the web context -->
 
-	 
-	<!-- Annotation based controllers -->
-	<bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping"/>

```

Comment:
@angshu that's for Core 2.4 compatibility and is backward compatible.

Cc @dkayiwa